src/js/utils/StringUtil.js

```diff
@@ -215,6 +215,16 @@ const StringUtil = {
     }, []);
 
     return this.idToTitle(splitID, splitBy, replace, removeConsecutive);
+  },
+
+  lrtrim(str, trimStr = " ") {
```

Comment:
I improved this. So now we include the default trim values as well as the one we would like to lrtrim. Also, included more test cases.